dashboards-observability/public/components/integrations/plugins/nginx/doc/constant.tsx

```diff
@@ -0,0 +1,168 @@
+  STEP2_DESC = 'In this step we need to provide the location of the access file.',
+  STEP3_DESC = 'In this step we need to update the fluent configuration file (fluent.conf)',
+  STEP4_DESC = 'In this step we need to provide the location of the access file.',
+  STEP5_DESC = 'In this setting we have created nginx_json_format as index to fetch data.',
+  CONFIGURATION_STEP1 = `'{'
+  '"msec": "$msec", ' # request unixtime in seconds with a milliseconds resolution
+  '"connection": "$connection", ' # connection serial number
+  '"connection_requests": "$connection_requests", ' # number of requests made in connection
+  '"pid": "$pid", ' # process pid
+  '"request_id": "$request_id", ' # the unique request id
+  '"request_length": "$request_length", ' # request length (including headers and body)
+  '"remote_addr": "$remote_addr", ' # client IP
+  '"remote_user": "$remote_user", ' # client HTTP username
+  '"remote_port": "$remote_port", ' # client port
+  '"time_local": "$time_local", '
+  '"time_iso8601": "$time_iso8601", ' # local time in the ISO 8601 standard format
+  '"request": "$request", ' # full path no arguments if the request
+  '"request_uri": "$request_uri", ' # full path and arguments if the request
+  '"args": "$args", ' # args
+  '"status": "$status", ' # response status code
+  '"body_bytes_sent": "$body_bytes_sent", ' # the number of body bytes exclude headers sent to a client
+  '"bytes_sent": "$bytes_sent", ' # the number of bytes sent to a client
+  '"http_referer": "$http_referer", ' # HTTP referer
+  '"http_user_agent": "$http_user_agent", ' # user agent
+  '"http_x_forwarded_for": "$http_x_forwarded_for", ' # http_x_forwarded_for
+  '"http_host": "$http_host", ' # the request Host: header
+  '"server_name": "$server_name", ' # the name of the vhost serving the request
+  '"request_time": "$request_time", ' # request processing time in seconds with msec resolution
+  '"upstream": "$upstream_addr", ' # upstream backend server for proxied requests
+  '"upstream_connect_time": "$upstream_connect_time", ' # upstream handshake time incl. TLS
+  '"upstream_header_time": "$upstream_header_time", ' # time spent receiving upstream headers
+  '"upstream_response_time": "$upstream_response_time", ' # time spent receiving upstream body
+  '"upstream_response_length": "$upstream_response_length", ' # upstream response length
+  '"upstream_cache_status": "$upstream_cache_status", ' # cache HIT/MISS where applicable
+  '"ssl_protocol": "$ssl_protocol", ' # TLS protocol
+  '"ssl_cipher": "$ssl_cipher", ' # TLS cipher
+  '"scheme": "$scheme", ' # http or https
+  '"request_method": "$request_method", ' # request method
+  '"server_protocol": "$server_protocol", ' # request protocol, like HTTP/1.1 or HTTP/2.0
+  '"pipe": "$pipe", ' # "p" if request was pipelined, "." otherwise
+  '"gzip_ratio": "$gzip_ratio"'
+  '}'`,
+  CONFIGURATION_STEP2 = `##this uses the our custom log format access_log /var/log/nginx/json_format.log custom_format`,

```

Comment:
I would suggest having separate files for the markdown text. It is easier to manage, because some documentation team can update that file without affecting the code. Refer the file from this code. You can have multiple files like step1.md, step2.md (or step-1.md to make it more dynamically accessible from the code using regex).